test/suites/ACID.m

```diff
@@ -319,6 +319,7 @@
 function [stat] = peaks_contour()
   stat.description = 'Test contour plots.';
   stat.unreliable = isMATLAB('<', [8,4]) || isOctave; %R2014a and older
+  % see #604; contour() produces inconsistent output
```

Comment:
Can you leave a `TODO` or `FIXME` there to make sure that we come back to those?